tests/storages_tests/rdb_tests/test_models.py

```diff
@@ -28,18 +28,23 @@ class TestStudySystemAttributeModel(object):
 
     @staticmethod
     def test_find_by_study_and_key(session):
+        # type: (Session) -> None
+
         study = StudyModel(study_id=1, study_name='test-study')
         session.add(StudySystemAttributeModel(study_id=study.study_id, key='sample-key',
                                               value_json='1'))
         session.commit()
 
-        assert '1' == StudySystemAttributeModel.find_by_study_and_key(study, 'sample-key',
```

Comment:
After adding the type hint `# type: (Session) -> None`, `mypy` emitted an error as follows:
```
test_models.py:38: error: Item "None" of "Optional[StudySystemAttributeModel]" has no attribute "value_json"
```

Hence, I rewrote this code to add `None` check.